rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -37,7 +45,7 @@ def _precompiled_apple_resource_bundle_impl(ctx):
     platform_prerequisites = platform_support.platform_prerequisites(
         apple_fragment = ctx.fragments.apple,
         config_vars = ctx.var,
-        device_families = ["iphone", "ipad"],
+        device_families = _DEVICE_FAMILIES_BY_PLATFORM_TYPE[platform_type],
```

Comment:
Could we instead just use: https://github.com/bazelbuild/rules_apple/blob/e12f3a5f445b08787f1e6594e3dcd7d7b64d80fe/apple/internal/platform_support.bzl#L131